lib/_stream_wrap.js

```diff
@@ -4,6 +4,7 @@ const assert = require('assert');
 const util = require('util');
 const Socket = require('net').Socket;
 const JSStream = process.binding('js_stream').JSStream;
+const Buffer = require('buffer').Buffer;
```

Comment:
It is available, but I thought that we decided to not use globals as much as we can.